jvm-packages/xgboost4j-tester/src/main/java/ml/dmlc/xgboost4j/tester/App.java

```diff
@@ -1,26 +0,0 @@
-package ml.dmlc.xgboost4j.tester;
-
-import ml.dmlc.xgboost4j.java.example.*;
-
-import java.io.IOException;
-import ml.dmlc.xgboost4j.java.XGBoostError;
-
-public class App {
-  public static void main(String[] args) throws IOException, XGBoostError {
```

Comment:
Any reason to delete xgboost4j-tester?

Reply:
I will bring back integration tests but using a different method.

Reply:
wow, expecting that.